src/swarm/neo/util/AcquiredResources.d

```diff
@@ -372,6 +372,167 @@ unittest
 
 /*******************************************************************************
 
+    Singleton (per-request) acquired resources of the templated type. An
```

Comment:
It seems that my broken English can't parse `Singleton acquired resources of the templated type.`, but I'm not sure if this is incorrect.